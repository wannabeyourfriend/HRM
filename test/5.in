1
3
inbox
outbox
jump 1