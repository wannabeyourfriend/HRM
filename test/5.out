Error on instruction 3